### ISSUE TITLE
Neovide ignores remember_window_size and confirm_quit on HEAD

### ISSUE THREAD

**Issue by 9mm** [neovide/neovide#1951]
**Describe the bug**

1. command+Q with unsaved changes immediately quits and ignores config option `vim.g.neovide_confirm_quit = true`

2. When using `vim.g.neovide_remember_window_size = true` I notice it restores the window to proper size about 1 in 20 times. I did notice another issue with no resolution mentioning window transparency, and I'm also using window transparency. Potentially due to this?

**To Reproduce**

To reproduce, install from head:

    brew install --HEAD neovide

Go to terminal and type `neovide`

Resize the window, and press command+Q

Reopen neovide repeatedly over and over with `neovide` from command line and the window will restore about 1 out of 20 times, which I would suppose is from a race condition

---

Regarding the other issue, open neovide. Type in the window and hit command+Q

**Expected behavior**

The window should resize exactly the same every time

Neovide should not allow closing 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS 13.4.1
 - Neovide Version 0.10.4
 - Neovim Version:
 
NVIM v0.9.1
Build type: Release
LuaJIT 2.1.0-beta3

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**

**Comment by 9mm** [comment]
Maybe there is more context here? https://github.com/neovide/neovide/issues/1580

**Comment by fredizzimo** [comment]
Yes, this is a duplicate of https://github.com/neovide/neovide/issues/1580
